Gruntfile.js

```diff
@@ -120,9 +120,25 @@ grunt.initConfig({
 			src: "build/report/lcov/lcov.info"
 		}
 	},
+	concurrent: {
+		target1: [
+			  "build"
+		],
+		target2: [
+			"jshint",
+			"jscs",
+			"search",
+			"qunit"
+		],
+		target3: [
+			"test-on-node"
```

Comment:
I changed the target names as suggested.  I tried having qunit as it's own target but it actually runs slower fr me.